mucommander-translator/src/main/resources/dictionary_en.properties

```diff
@@ -484,7 +484,8 @@ search_dialog.max_threads = Maximum
 search_dialog.case_sensitive = $[file_selection_dialog.case_sensitive]
 search_dialog.matches_regexp = $[file_selection_dialog.matches_regexp]
 search_dialog.wildcards = (* = any string, ? = any character, \\ = escape for literals: *?)
-search_dialog.search_text  = Containing text
+search_dialog.search_text = Containing text
+search_dialog.search_text_optional = Text search (Optional)
```

Comment:
why here and not in dictionary.properties?

Reply:
right, I've updated PR